utils/cmd/internal/get_latest_version_commands.go

```diff
@@ -0,0 +1,30 @@
+package internal
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/docker/test_env"
+)
+
+var GetLatestEthereumClientVersionCmd = &cobra.Command{
+	Use:   "get-latest-ethereum-client-version",
+	Short: "Get the latest Ethereum client release version from Github",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return fmt.Errorf("please provide the repository in the format 'owner/repo:tag'")
```

Comment:
owner or org?

Reply:
@Tofel fixed in a8fd77b9d859ec897a73fafdf5865fe811f64d1c